besu/src/test/java/org/hyperledger/besu/cli/subcommands/storage/TrieLogHelperTest.java

```diff
@@ -58,7 +61,7 @@ class TrieLogHelperTest {
 
   private static final StorageProvider storageProvider = new InMemoryKeyValueStorageProvider();
   private static BonsaiWorldStateKeyValueStorage inMemoryWorldState;
-  private TrieLogHelper trieLogHelper;
+  private TrieLogHelper nonValidatingTrieLogHelper;
```

Comment:
what you do you mean with nonValidatingTrieLogHelper?

Reply:
The validation enforces 512 minimum so I'm skipping it for the sake of the unit tests not having to create lots of data.